### PR TITLE
update AC support policies

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1529

Will add the same changes to Nebula once they're approved here.

### CODE REVIEW

File: software/ac-support-policy.md [hunk 1]
```diff
@@ -140,16 +140,17 @@ The following tables list the lifecycles for each published version of Astronome
...
 
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or [migrating to Astro Runtime 5.0](migrate-to-runtime.md) to receive long term support for Apache Airflow 2.3.
+
 ### LTS releases
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)¹       | May 21, 2021   | November 2022           |
```
Comment (jwitz) [inline software/ac-support-policy.md:153]:
```suggestion
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md).       | May 21, 2021   | November 2022           |
```

File: software_versioned_docs/version-0.25/ac-support-policy.md [hunk 2]
```diff
@@ -118,22 +118,21 @@ Maintenance is discontinued the last day of the month for a given version. For e
...
+### Stable releases
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)       | Oct 12, 2021   | April 2022              |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)¹       | May 21, 2021   | November 2022           |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2022            |
```
Comment (jwitz) [inline software_versioned_docs/version-0.25/ac-support-policy.md:126]:
```suggestion
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
```

File: software_versioned_docs/version-0.29/ac-support-policy.md [hunk 3]
```diff
@@ -126,17 +126,16 @@ The following tables list the lifecycle for each published version of Astronomer
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)¹       | May 21, 2021   | November 2022           |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2022            |
```
Comment (jwitz) [inline software_versioned_docs/version-0.29/ac-support-policy.md:130]:
```suggestion
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)      | May 21, 2021   | November 2022           |
| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
```

File: software_versioned_docs/version-0.29/ac-support-policy.md [hunk 4]
```diff
@@ -126,17 +126,16 @@ The following tables list the lifecycle for each published version of Astronomer
...
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or migrating to Astro Runtime 5.0.x to receive long term support for Apache Airflow 2.3.
+
 ### LTS Releases
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)¹       | May 21, 2021   | November 2022           |
```
Comment (jwitz) [inline software_versioned_docs/version-0.29/ac-support-policy.md:139]:
```suggestion
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)      | May 21, 2021   | November 2022           |
```

File: software_versioned_docs/version-0.28/ac-support-policy.md [hunk 5]
```diff
@@ -122,18 +122,16 @@ The following tables contain the exact lifecycle for each published version of A
...
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
 | [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)       | Oct 12, 2021   | April 2022              |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
 | [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |
 
+> ¹ In November 2022, Astronomer Certified 2.3 was reclassified as a stable release only.
```
Comment (jwitz) [inline software_versioned_docs/version-0.28/ac-support-policy.md:129]: @paolaperaza Is there any recommendation we want to give here for Software versions where Runtime isn't yet supported?

File: software/ac-support-policy.md [hunk 6]
```diff
@@ -141,15 +141,16 @@ The following tables list the lifecycles for each published version of Astronome
...
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
 | [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |
 
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or [migrating to Astro Runtime 5.0](migrate-to-runtime.md) to receive long term support for Apache Airflow 2.3.
+
 ### LTS releases
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
 | [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
```
Comment (jwitz) [inline software/ac-support-policy.md:153]: @paolaperaza Am I reading this right that there will be no LTS certified releases at the end of this month?

File: software_versioned_docs/version-0.29/ac-support-policy.md [hunk 7]
```diff
@@ -126,17 +126,16 @@ The following tables list the lifecycle for each published version of Astronomer
...
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
-| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)      | May 21, 2021   | November 2022           |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
 | [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |
 
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or migrating to Astro Runtime 5.0.x to receive long term support for Apache Airflow 2.3.
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.29/ac-support-policy.md:133]:
```suggestion
> ¹ In November 2022, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or migrating to Astro Runtime 5.0.x to maintain Apache Airflow support.
```

File: software_versioned_docs/version-0.25/ac-support-policy.md [hunk 8]
```diff
@@ -118,22 +118,21 @@ Maintenance is discontinued the last day of the month for a given version. For e
...
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
 | [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)       | Oct 12, 2021   | April 2022              |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
 | [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |
 
-### LTS Releases
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only.
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/ac-support-policy.md:129]:
```suggestion
> ¹ In November 2022, Astronomer Certified 2.3 was reclassified as a stable release only.
```

File: software_versioned_docs/version-0.25/ac-support-policy.md [hunk 9]
```diff
@@ -118,22 +118,21 @@ Maintenance is discontinued the last day of the month for a given version. For e
...
-### Stable Releases
+### Stable releases
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
 | [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
-| [2.2](https://github.com/astronomer/ap-airflow/blob/master/2.2.0/CHANGELOG.md)       | Oct 12, 2021   | April 2022              |
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/ac-support-policy.md:126]:
```suggestion
| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
```
Are the dates correct?

File: software/ac-support-policy.md [hunk 10]
```diff
@@ -141,15 +141,16 @@ The following tables list the lifecycles for each published version of Astronome
...
-| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
+| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)¹       | April 30, 2022 | October 2022            |
 | [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |
 
+> ¹ In November 2023, Astronomer Certified 2.3 was reclassified as a stable release only. Astronomer recommends upgrading or [migrating to Astro Runtime 5.0](migrate-to-runtime.md) to receive long term support for Apache Airflow 2.3.
```
Comment (kwcanuck) [inline software/ac-support-policy.md:147]:
```suggestion
> ¹ In November 2022, Astronomer Certified 2.3 was reclassified as a stable release only. To maintain Apache Airflow support, Astronomer recommends upgrading or [migrating to Astro Runtime 5.0](migrate-to-runtime.md).
```
It's not just Apache Airflow 2.3 support, is it? It's future versions as well? If not:

To maintain Apache Airflow 2.3 support, Astronomer recommends upgrading or [migrating to Astro Runtime 5.0](migrate-to-runtime.md).

**Review by kwcanuck — APPROVED** [review]

File: software/ac-support-policy.md (hunk 10 above)
Comment (jwitz) [reply software/ac-support-policy.md:147]: It usually would be, but the problem for users right now is "I just lost long term support of 2.3 and I need to get it back"

File: software_versioned_docs/version-0.25/ac-support-policy.md (hunk 9 above)
Comment (jwitz) [reply software_versioned_docs/version-0.25/ac-support-policy.md:126]: Yes unfortunately - this release had its support timeline shortened

File: software_versioned_docs/version-0.28/ac-support-policy.md (hunk 5 above)
Comment (paolaperaza) [reply software_versioned_docs/version-0.28/ac-support-policy.md:129]: Good question. I think we just tell them to contact their customer success manager?

File: software_versioned_docs/version-0.28/ac-support-policy.md [hunk 11]
```diff
@@ -122,18 +122,16 @@ The following tables contain the exact lifecycle for each published version of A
 
 | AC Version                                                                           | Release Date   | End of Maintenance Date |
 | ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
-| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.28/ac-support-policy.md:125]: Do we see a world in which we have an "archive" of previously supported versions?

**Review by paolaperaza — APPROVED** [review]
This LGTM -- the note on reclassifying AC is well framed

**Pull request merged**